modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -269,3 +269,12 @@ channels:
 |Boolean. The default value is `false`.
 
 |===
+
+[NOTE]
+====
+In certain cases error occurs when the `minVersion` and `maxVersion` properties are used to filter for a specific version range for an operator in the `ImageSetConfiguration` parameters. The error message will state that there are `multiple channel heads`. This is because when the filter is applied, the update graph of the operator is truncated.
```

Comment:
>In certain cases

Not sure we need this phrase. It doesn't help much and it makes me ask Which cases? 

Using the the `minVersion` and `maxVersion` properties to filter for a specific Operator version range can result in a `multiple channel heads` error.

Reply:
Ack.